tests/test_counting_hash.py

```diff
@@ -35,6 +38,10 @@ def teardown():
     utils.cleanup()
 
 
+def scriptpath(script):
+    return script
```

Comment:
Correct. If you move code from one file to another and you reference a
function from the original file then you should continue to reference that
function from the new location instead of writing a "do nothing" stub.

On Fri, Mar 27, 2015, 21:21 Sherine <notifications@github.com> wrote:

> In tests/test_counting_hash.py
> <https://github.com/ged-lab/khmer/pull/809#discussion_r27323663>:
>
> > @@ -35,6 +38,10 @@ def teardown():
> >      utils.cleanup()
> >
> >
> > +def scriptpath(script):
> > +    return script
>
> and if I used script = 'load-into-counting.py' works locally only !
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/ged-lab/khmer/pull/809/files#r27323663>.
>